### PR TITLE
LOC CHECKIN | microsoft/vstest master | 20200526

### PR DESCRIPTION
**Pull request by v-chmart**
## Description
Localization update only

## Related issue
Kindly link any related issues. E.g. Fixes #xyz.

### CODE REVIEW

File: src/Microsoft.TestPlatform.Extensions.BlameDataCollector/Resources/xlf/Resources.cs.xlf [hunk 1]
```diff
@@ -62,7 +62,7 @@
       </trans-unit>
       <trans-unit id="UnexpectedValueForInactivityTimespanValue">
         <source>Unexpected value '{0}' provided as timeout. Please provide a value in this format: 1.5h / 90m / 5400s / 5400000ms / 5400000</source>
-        <target state="new">Neočekávaná hodnota {0} zadaná pro ExpectedExecutionTimeOfLongestRunningTestInMinutes. Zadejte jako vstup kladné celé číslo.</target>
+        <target state="new">Unexpected value '{0}' provided as timeout. Please provide a value in this format: 1.5h / 90m / 5400s / 5400000ms / 5400000</target>
```
Comment (nohwnd) [inline src/Microsoft.TestPlatform.Extensions.BlameDataCollector/Resources/xlf/Resources.cs.xlf:65]: Those strings are not translated, is that expected?

Comment (janbaltus) [reply src/Microsoft.TestPlatform.Extensions.BlameDataCollector/Resources/xlf/Resources.cs.xlf:65]: String state is new and we will cover them in next  delivery.

Comment (janbaltus) [reply src/Microsoft.TestPlatform.Extensions.BlameDataCollector/Resources/xlf/Resources.cs.xlf]: PR updated with fully loclaized xlf

**Comment by nohwnd** [comment]
Perfect, I was just trying to understand how to read this next time. Now I saw the new state. In the original commit. Thanks!

**Review by nohwnd — APPROVED** [review]

**Pull request merged**